kss/builder/decanter/kss-assets/css/kss.css

```diff
@@ -410,7 +410,7 @@ sub {
   #kss-node > .su-brand-bar {
     position: fixed;
     top: 0;
-    z-index: 9999; }
+    z-index: 22222; }
```

Comment:
Wowza.

Reply:
lol overkill?

Reply:
Btw, @sherakama, @JBCSU is helping me with some JS refactoring so please hold off on reviewing the JS for now. Thanks!